zrml/prediction-markets/src/lib.rs

```diff
@@ -2613,6 +2620,10 @@ mod pallet {
             T::PalletId::get().0
         }
 
+        fn market_account(market_id: MarketIdOf<T>) -> T::AccountId {
+            T::PalletId::get().into_sub_account_truncating(market_id.saturated_into::<u128>())
+        }
+
```

Comment:
Why did you add this again?

```suggestion
```

Reply:
I think that now we have tight coupling we may perhaps able to remove `PredictionMarketsPalletId` from market_commons Config. however if we want to keep `market_account()` in the trait then I accept your suggested edit.

Reply:
but I think we can't get rid of `PredictionMarketsPalletId` as we can't access prediction_market config in matket_commons.